### ISSUE TITLE
 What to look for in an accessibility vendor

### ISSUE THREAD

**Issue by ljgoslin** [a11yproject/a11yproject.com#724]
[Join our live webinar](https://accessibility.deque.com/what-to-look-for-in-an-accessibility-vendor?hsCtaTracking=4a4292ff-4750-4a51-875d-07870402e873%7C96b89eae-ef33-4abd-a691-8c355fd19695): Reference and apply these key business requirements when evaluating accessibility vendors. January 29th, 1:00-1:30PM EST.

Without accessibility expertise, evaluating accessibility vendors can be a difficult task. Who can you trust?

**Sachin Gupta**, Director of Development Services at Deque, will review key characteristics organizations should evaluate when looking for an accessibility partner and basics they should have in mind when evaluating tools and services. In this webinar, you will learn:

- The types of services and tools available across the accessibility market.
- How your organization’s accessibility maturity affects the buying process.
- The average commitment length of common accessibility service engagements.
- The importance of expertise and subject matters experts in the context of remediation, accessibility audits, and training.

**Comment by ericwbailey** [comment]
Closed via https://github.com/a11yproject/a11yproject.com/commit/429bd3070f9fed6d01ca80ed94882a844658790e.
